openhtf/core/test_state.py

```diff
@@ -68,6 +68,26 @@ class DuplicateAttachmentError(Exception):
   """Raised when two attachments are attached with the same name."""
 
 
+class Subscribable(object):
+  """Subscribable object."""
```

Comment:
A _slightly_ more descriptive docstring would be nice. Maybe a few words about why this exists.